web/src/components/storage/index.js

```diff
@@ -23,6 +23,7 @@ export { default as ProposalPage } from "./ProposalPage";
 export { default as ProposalPageMenu } from "./ProposalPageMenu";
 export { default as ProposalSettingsSection } from "./ProposalSettingsSection";
 export { default as ProposalSpacePolicySection } from "./ProposalSpacePolicySection";
+export { default as ProposalDeviceSection } from "./ProposalDeviceSection";
```

Comment:
I start wondering about the convenience of exporting this _internal_ components. We are doing so just for importing them in the test file (!!), but we can do it without polluting the index file.

I.e., 

```js
import ProposalDeviceSection from "~/components/storage/ProposalDeviceSection";
```

instead of 

```js
import { ProposalDeviceSection } from "~/components/storage";
```

Reply:
I agree. For example, I would say that *components/storage/index.js* should export the pages and the components that might be reused from pages out of storage. In that way the index file helps to know what is "public" in the storage namespace.

Reply:
> I agree. For example, I would say that _components/storage/index.js_ should export the pages and the components that might be reused from pages out of storage. In that way the index file helps to know what is "public" in the storage namespace.

And this is extensible to all namespaces.